AquaSecurity/AquaSecurity-Dashboard.json

```diff
@@ -0,0 +1,1050 @@
+{
+  "type":"Report",
+  "name":"Aqua Security",
+  "description":"",
+  "columns":[""],
+  "detailLevel":0,
+  "properties":"{\"layoutType\":\"gridster\"}",
+  "panels":[{
+    "type":"ReportPanel",
+    "name":"Detected Events",
+    "viewerType":"svv",
+    "detailLevel":2,
+    "queryString":" _sourceCategory=\"Audit\"\n| json field=_raw \"result\" | where %result = 3 | count by result",
```

Comment:
@jerbia Thanks for the PR, 
We are working on ensuring that these are easier/standard for users to deploy. 
Would you make 2 small changes please and i'll merge this PR?
In the JSON, the "scope" is defined as `_sourceCategory=\"Audit\"\n`, which I assume is the sourcecategory you used for the logs this dashboard was created against. 

Would you please change the scope to something like `$$aquaauditlogs` and 

secondly, Would you make a note in the README.md that `$$aquaauditlogs` must be replaced with something like `_sourceCategory=<source_category>\n` where <source_category> is the _sourceCategory for your aqua audit logs`?

Reply:
Hi. What is the $$aquaauditlogs? Is it an environment variable? 
How would a user provide a value to this environment variable? Is there some system setting in SumoLogic where you can provide values to environment variables?